qpc/utils.py

```diff
@@ -384,6 +384,8 @@ def log_request_info(method, command, url, response_json, response_code):
     :param response_json: the response returned from the request
     :param response_code: the status code being returned (ie. 200)
     """
+    if isinstance(response_json, (bytes, bytearray)):
+        response_json = response_json.decode("utf-8", errors="ignore")
```

Comment:
some responses won't make sense. even decoded.

Have you tested this with insights tarball and/or "all reports" tarball?